standard-dev.cfg

```diff
@@ -0,0 +1,45 @@
+# Standard development config to be used in standardized policies.
```

Comment:
We have a pretty good `README` at the moment, instead (or in addition to) documenting here i'd consider moving this to the `README`.